src/app.yaml

```diff
@@ -12,6 +12,8 @@ handlers:
   upload: favicon\.ico
 - url: /tests.*
   script: lib.gaeunit.gaeunit.tester
+- url: /_ah/spi/.*

```

Comment:
no, this one is OK: https://cloud.google.com/appengine/docs/python/endpoints/api_server